diff --git a/node_modules/fabric/dist/fabric.js b/node_modules/fabric/dist/fabric.js
index faee7fc..e90594d 100644
--- a/node_modules/fabric/dist/fabric.js
+++ b/node_modules/fabric/dist/fabric.js
@@ -21,7 +21,7 @@ if (typeof document !== 'undefined' && typeof window !== 'undefined') {
 }
 else {
   // assume we're running under node.js when document/window are not present
-  var jsdom = require('jsdom');
+  var jsdom = {};
   var virtualWindow = new jsdom.JSDOM(
     decodeURIComponent('%3C!DOCTYPE%20html%3E%3Chtml%3E%3Chead%3E%3C%2Fhead%3E%3Cbody%3E%3C%2Fbody%3E%3C%2Fhtml%3E'),
     {
@@ -31,8 +31,8 @@ else {
       resources: 'usable'
     }).window;
   fabric.document = virtualWindow.document;
-  fabric.jsdomImplForWrapper = require('jsdom/lib/jsdom/living/generated/utils').implForWrapper;
-  fabric.nodeCanvas = require('jsdom/lib/jsdom/utils').Canvas;
+  fabric.jsdomImplForWrapper = null;
+  fabric.nodeCanvas = null;
   fabric.window = virtualWindow;
   DOMParser = fabric.window.DOMParser;
 }
